client/python/samples/README.md

```diff
@@ -183,22 +187,21 @@ Image images/golden_retriever.jpeg has been classified as golden retriever with
 
 ### Vehicle detection model
 ```
-mkdir -p <PATH_TO_MODELS>/vehicle-detection/1
-cd <PATH_TO_MODELS>/vehicle-detection/1
-wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/vehicle-detection-0202/FP32/vehicle-detection-0202.xml
-wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/vehicle-detection-0202/FP32/vehicle-detection-0202.bin
+mkdir -p models
+curl --create-dirs https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/vehicle-detection-0202/FP32/vehicle-detection-0202.xml -o ${PWD}/models/vehicle-detection/1/vehicle-detection-0202.xml
+curl --create-dirs https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/vehicle-detection-0202/FP32/vehicle-detection-0202.bin -o ${PWD}/models/vehicle-detection/1/vehicle-detection-0202.bin
+chmod -R 755 ${PWD}/models/vehicle-detection
 ```
-For numeric data format OVMS can be started using a command:
+OVMS container can be started using a command:
 ```bash
-docker run -d --rm -v <PATH_TO_MODELS>/vehicle-detection:/models/vehicle-detection -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name vehicle-detection --model_path /models/vehicle-detection --port 9000 --rest_port 8000
+docker run -d --rm -v ${PWD}/models/vehicle-detection:/models/vehicle-detection -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name vehicle-detection --model_path /models/vehicle-detection --port 9000 --rest_port 8000 --layout NHWC
 ```
 
 
 ### Detect vehicle  <a name="detect-vehicle">
 
 
 #### **Make vehicle detection prediction using images in binary format:**
-#### **Required Model Server running with vehicle detection model:**
```

Comment:
I think this information is crucial.

Reply:
in the previous step is instruction how to start ovms with vehicle detection model, running prediction is the next step